pages/linux/at.md

```diff
@@ -13,3 +13,7 @@
 - Execute a script at the given time:
 
 `at {{hh:mm:ss}} -f {{path/to/file}}`
+
+- Reboot at 11pm on February 18th:
+
+`echo "reboot" | at {{11pm}} {{Feb 18}}`
```

Comment:
Wouldn't this need to be `sudo`?
```suggestion
`echo "reboot" | sudo at {{11pm}} {{Feb 18}}`
```

Reply:
Perhaps putting the braces around `reboot` is not needed, since it doesn't say "execute a command `reboot`", but just "reboot".

Reply:
* `reboot` really *is* a shell command (thanks for fixing up my markup!)
* `sudo` might be needed depending on whether one is root or not. I'd think it's self evident that the `reboot` command (and thus the `at` command) needs to be executed as root, so I'd not add the example with `sudo`

Reply:
Right. Note that your markup isn't actually fixed - the above is just a suggestion that you need to agree to by clicking the "commit suggestion" button.

I've updated my suggestion above to remove the `sudo` and make the command more generic. Perhaps we could alter the description for this command too?

My concern is that `reboot` is a specific command that does require `sudo`, so the example wouldn't necessarily work out-of-the-box - depending on who the reader is logged into their terminal as. It is for this reason that I'd prefer a command that does not require `sudo` privileges for the example.

Reply:
Rebooting a machine on off-office hours is by far the most common use for `at` I have and I suspect it might be quite common. So I think it makes sense to use "reboot" as an example.

Reply:
I see, @tpo. I'm still concerned that the example might not work for some users though. For example, if I'm logged in as a regular user and copy-paste the command (as a beginner might do), it will fail because the `at` command was not preceded by `sudo`.

Reply:
See https://github.com/tpo/tldr/commit/06b6cae8570c6f0d9de2aa1f1363bc2f676e8702

You are supposing that making that `sudo` will somehow fail less. I think that belief is unsubstanciated, as sudo isn't available by default on Linux systems and setting up sudo is not trivial. So I suggest to just mention the fact that one needs to be root.

Reply:
> sudo isn't available by default on Linux systems

Erm what kind of Linux system are you using? On Ubuntu, Arch, Centos, and Fedora it's definitely installed and configured by default as far as I can tell.

Thanks for the edit - I guess that could be ok for now (though another alternative to `reboot` might be simply `{{command}}`, which would be consistent with some other pages.

Reply:
I'm on Debian.

We have already discussed `{{command}}`.